message tab_ChooseAttackerTable 
{
	optional string tname = 1;
	repeated tab_ChooseAttacker tlist = 2;
}

message tab_ChooseAttacker
{
	optional int32 id = 1;
	optional string Comment = 2;
	optional int32 Level = 3;
	optional int32 NeedCoins = 4;
}